bedrock/firefox/templates/firefox/new/batm/machine-a.html

```diff
@@ -0,0 +1,73 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-new-batm-anim', ['en-US']) %}
+    {% javascript 'experiment_firefox_new_batm_anim' %}
```

Comment:
I believe the idea is for the ads to link explicitly to `xv=batma` and Traffic Cop would then serve up of three variants. But if they can do that kind of randomization on the ad server, then we wouldn't need Traffic Cop at all. I'll check with Fabio.